azext_edge/edge/_help.py

```diff
@@ -698,86 +698,31 @@ def load_iotops_help():
     ] = """
         type: command
         short-summary: Create an asset.
-        long-summary: |
-                      Either custom location or cluster name must be provided. This command will check
-                      for the existance of the associated custom location and cluster and ensure that
-                      both are set up correctly with the microsoft.deviceregistry.assets extension.
-
-                      At least one data point or event must be defined during asset creation. For examples
-                      of file formats, please see aka.ms/aziotops-assets
+        long-summary: For examples of file formats, please see aka.ms/aziotops-assets
```

Comment:
only examples mentioned in long summary is this intended?

Reply:
everything else does not apply anymore